pkg/apisix/cluster.go

```diff
@@ -310,7 +310,11 @@ func (s *cluster) listResource(ctx context.Context, url string) (*listResponse,
 }
 
 func (s *cluster) createResource(ctx context.Context, url string, body io.Reader) (*createResponse, error) {
-	req, err := http.NewRequestWithContext(ctx, http.MethodPut, url, body)
+	return s.createResourceWithMethod(ctx, url, body, http.MethodPut)
```

Comment:
Why encapsulating it further by `s.createResourceWithMethod`, I didn't find any other places to call the `s.createResourceWithMethod`.

Reply:
`createResourceWithMethod` removed